crates/turbo-malloc/Cargo.toml

```diff
@@ -9,8 +9,8 @@ autobenches = false
 [lib]
 bench = false
 
-[target.'cfg(not(target_os = "linux"))'.dependencies]
-mimalloc = { version = "0.1.32", features = [], optional = true }
+[dependencies]
```

Comment:
I think this is more correct with the alternative below?

Reply:
Not sure about this one. cc @kwonoj @Brooooooklyn